client/store/cartReducer.js

```diff
@@ -2,23 +2,30 @@ import axios from 'axios'
 
 const SET_CART = 'SET_CART'
 
+const _setCart = (cartData) =>{
+    return {
+        type: SET_CART,
+        cartData
+    }
+}
 
 export const fetchCart = (user) =>{
     return async (dispatch) =>{
         console.log('fetched cart', user)
-        if(user){
-            const { data } = await axios.get(`/api/orders/5`)
-            console.log('we fetched cart' , data)
+        try{
+            const { data } = await axios.get(`/api/orders/${user.id}`)
```

Comment:
I pointed this out on the route, but this axios call is fetching orders for user.id, but the route is querying the DB for the order with a primary key.

The primary key of an order is not the user.id, because a user can have many orders, only one of which should have a status of 'cart'

Or maybe a user can have multiple carts? Unsure.